test/models/item_test.rb

```diff
@@ -4,7 +4,7 @@ class ItemTest < ActiveSupport::TestCase
   attr_reader :item
 
   def setup
-    @item = Item.create(name: 'espresso')
+    @item = Item.create(title: 'espresso')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.